### ISSUE TITLE
Issue with 'Switch to Basic HTML Version' when using the Search on top of the CDTS template

### ISSUE THREAD

**Issue by rogercote8** [wet-boew/cdts-sgdc#409]
We are using the CDTS template for our app.  Latest online version.  We are mostly using Edge as a browser.
When using the search box on top, the link to switch to Basic HTML version is not working.  We notice that the rendering looks like the following:

![image](https://user-images.githubusercontent.com/76535263/152042728-c1e44b59-fb81-44f3-971b-0784899f3cf9.png)

Clicking on the link does not switch to basic HTML.  It appears that the issue is that &#38; is not interpreted as &.

![image](https://user-images.githubusercontent.com/76535263/152042904-82237289-e851-4463-981b-797b28cf4634.png)

**Comment by rogercote8** [comment]
We are working with a Power Apps Portal from Microsoft.  Power Apps is using the URL parameter to get to specific page.   Here is an example of a URL:
https://rhrrp-pdrp-dev.powerappsportals.com/en/rh-projects-list/arr/submit/?id=dc39cd51-747f-ec11-8d21-000d3a0c7445

Now when loading the page, the wbdisable parameter is added to the URL as follow.  It is adding an extra parameter using &#38; to separate the parameter making it ineffective when clicking on it.
***@***.***

When clicking on the URL, you get the following:
***@***.***
If I replace the &#38; with & only, it works because I get the following:
***@***.***

This has nothing to do with the Search after all.

Roger Côté
(Il/lui – He/him/his)

Conseiller technique
Solutions numériques d’entreprise
Direction générale de l’innovation, de l’information et de la technologie (DGIIT)
Emploi et Développement social Canada (ESDC) / Gouvernement du Canada
***@***.******@***.***> / Tél. :

Technical Advisor
Enterprise Digital Solutions
Information, Innovation and Technology Branch (IITB)
Employment and Social Development Canada (ESDC) / Government of Canada
***@***.******@***.***> / Tel:

**Comment by ahmad-shahid** [comment]
We were able to reproduce the issue. The fix is ready. It will be deployed by next week.

**Comment by rogercote8** [comment]
Super, thanks.

**Comment by ahmad-shahid** [comment]
Update of run version to v4.0.44 has been postponed because of unrelated problems. Once the deployment is done, I will provide an update here.

**Comment by ahmad-shahid** [comment]
This was done and is part of the current RUN version release. Closing the issue.
